sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -265,6 +265,23 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     }
   }
 
+  test("Describe Table") {
```

Comment:
This test case is not needed. `SQLQueryTestSuite` already contains it after you regenerate the result file.